src/Helper/ArrayHelper.php

```
@@ -37,7 +37,18 @@ public static function getByPath($data, $arguments)
 			return null;
 		}
 
-		$args = is_array($arguments) ? $arguments : explode('.', $arguments);
+		if (is_array($arguments))
+		{
+			$args = $arguments;
+		}
+		elseif (is_string($arguments))
+		{
+			$args = explode('.', $arguments);
+		}
+		else
+		{
+			return null;
```

Maybe throw `InvalidArgumentException`?